editor/svgcanvas.js

```diff
@@ -554,12 +554,14 @@ var getIntersectionList = this.getIntersectionList = function(rect) {
 	}
 	
 	var resultList = null;
-	if (typeof(svgroot.getIntersectionList) == 'function') {
-		// Offset the bbox of the rubber box by the offset of the svgcontent element.
-		rubberBBox.x += parseInt(svgcontent.getAttribute('x'), 10);
-		rubberBBox.y += parseInt(svgcontent.getAttribute('y'), 10);
+	if (!svgedit.browser.isIE) {
+		if (typeof(svgroot.getIntersectionList) == 'function') {
+			// Offset the bbox of the rubber box by the offset of the svgcontent element.
+			rubberBBox.x += parseInt(svgcontent.getAttribute('x'), 10);
+			rubberBBox.y += parseInt(svgcontent.getAttribute('y'), 10);
```

Comment:
I wonder whether the root of the problem is modifying `x`/`y` in place. At least that's what the error seems to imply.

Perhaps instead `svgcontent.createSVGRect` should be invoked here similarly to line 553 above?